src/situations/commun/vues/defi/champ_saisie.vue

```diff
@@ -46,6 +47,15 @@ export default {
 
   components: { BoutonLecture },
 
+  mounted() {
+    this.$watch(
+      "$refs.boutonLecture.joueSon",
+      (value) => {
+        return this.enregistreActiviteSon(value);
+      }
+    );
+  },
+
```

Comment:
J'ai suivi l'article ci-dessous pour me permettre de surveiller le changement de valeur de la propriété JoueSon du composant enfant bouton lecture depuis le parent champ de saisie. https://stackoverflow.com/questions/51225378/how-to-watch-child-properties-changes-from-parent-component